terraform/main.tf

```diff
@@ -1,12 +1,13 @@
 // Make sure you export the following environment variables!
 //    export AWS_ACCESS_KEY_ID="access-key"
- //   export AWS_SECRET_ACCESS_KEY="secret-key"
- //   export AWS_DEFAULT_REGION="region"
+//   export AWS_SECRET_ACCESS_KEY="secret-key"
+//   export AWS_DEFAULT_REGION="region"
```

Comment:
Align the exports. Also, why not put the region into the `provider` block? Then you could make it a terraform variable. The only reason the access key isn't a variable is to discourage people from saving it on disk